build_tools/waf_dynamo.py

```diff
@@ -284,19 +278,19 @@ def apply_framework(self):
     for x in self.to_list(self.env['FRAMEWORKPATH']):
         frameworkpath_st='-F%s'
         self.env.append_unique('CXXFLAGS',frameworkpath_st%x)
-        self.env.append_unique('CCFLAGS',frameworkpath_st%x)
+        self.env.append_unique('CFLAGS',frameworkpath_st%x)
```

Comment:
Annoying change, but there's no cc / CCFLAGS etc. Everything is CXX* or just C* in waf now.